ansible/roles/system/tasks/main.yml

```diff
@@ -252,6 +252,11 @@
       - gpio
       - dialout
 
+- name: Add the necessary kernel modules using `modprobe`.
+  ansible.builtin.shell: |
+    sudo modprobe veth
+    sudo modprobe ip_tables
```

Comment:
Since a reboot is really required for the Docker containers to start successfully, we won't be needing these anymore.